src/test/suite/mdbExtensionController.test.ts

```diff
@@ -7,15 +7,15 @@ import { CollectionTreeItem } from '../../explorer';
 import { VIEW_COLLECTION_SCHEME } from '../../editors/collectionDocumentsProvider';
 import ConnectionTreeItem from '../../explorer/connectionTreeItem';
 import DatabaseTreeItem from '../../explorer/databaseTreeItem';
-import { CollectionTypes } from '../../explorer/collectionTreeItem';
+import { CollectionTypes } from '../../explorer/documentListTreeItem';
 import { mdbTestExtension } from './stubbableMdbExtension';
 import ConnectionController from '../../connectionController';
 import { StorageController } from '../../storage';
 
 const testDatabaseURI = 'mongodb://localhost:27018';
 
 suite('MDBExtensionController Test Suite', () => {
-  afterEach(function() {
+  afterEach(function () {
```

Comment:
Maybe use an arrow function? we don't care about context here i believe.

Reply:
Do we actually want to get rid of these old-fashioned function keywords everywhere? ;)

Reply:
good call